.github/workflows/update-specs-and-client-libraries.yaml

```diff
@@ -144,6 +150,12 @@ jobs:
               poetry lock
             version: ${{ needs.generate_specs_and_libraries.outputs.python_version }}
             update: ${{ inputs.update-onfido-python }}
+          - generator: ruby/faraday
+            git_repo_id: onfido-ruby
+            preCommit: |
+              bundle lock --update
+            version: ${{ needs.generate_specs_and_libraries.outputs.ruby_version }}
+            update: ${{ inputs.update-onfido-ruby || true}}
```

Comment:
This `|| true` should remain here only while testing CI

Reply:
i've removed it.

Reply:
It looks it's still there... 🤔

Reply:
forgot to add the file, removed now.